esbuild.dev.config.ts

```diff
@@ -0,0 +1,31 @@
+const esbuild = require("esbuild");
+const inlineImage = require("esbuild-plugin-inline-image");
+const { sassPlugin } = require("esbuild-sass-plugin");
+const postcss = require('postcss');
+const autoprefixer = require('autoprefixer');
+const { dependencies } = require("./package.json");
+
+esbuild
+  .serve(
+    {
+      servedir: "public",
+      port: 8000,
```

Comment:
see if we can create a `.env` file and get this port value from there as there could be chances on some machines that` 8000` is occupied and it doesn't run
also, keep `8000` as default in case the variable from `.env` is not loaded/undefined